config.yaml

```diff
@@ -225,6 +225,23 @@ teams:
       - hartm
       - minyu66
       - thelinuxfoundation
+  - name: hiero-mirror-node-committers
+    maintainers:
+      - steven-sheehy
+      - xin-hedera
+    members:
+      - bilyana-gospodinova
+      - edwin-greene
+      - IvanKavaldzhiev
+      - jnels124
+      - kselveliev
+      - mgoelswirlds
+      - nickeynikolovv
+  - name: hiero-mirror-node-maintainers
+    maintainers:
+      - steven-sheehy
+    members:
+      - xin-hedera
```

Comment:
My assumption is that if you're a maintainer of a group you're implicitly a member as well? I can't find it documented anywhere.

Reply:
Yes, that is my assumption as well.

Reply:
that is correct.